packages/googlecloud/functions/getanswer/inquirer.py

```diff
@@ -234,7 +234,7 @@ def process_and_concat_documents(retrieved_docs):
     return combined_content, original_documents
 
 
-def get_indepth_response_from_query(df, db_fc, db_cj, db_pdf, db_pc, db_news, query, k):
+def get_indepth_response_from_query(df, db_fc, db_cj, db_pdf, db_pc, db_news, query, k, return_context):
```

Comment:
return_context flag: will return the combined_docs_content from the FAISS search back up from the getanswer query, so it can be fed back into the evaluate functions for all of the retrieval context metrics